recipes/viral_consensus/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# update environment variables
+export CFLAGS="$CFLAGS -I$PREFIX/include"
+export CPPFLAGS="$CPPFLAGS -I$PREFIX/include"
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
+export CPATH=${PREFIX}/include
+export INCLUDES="-I${PREFIX}/include"
+export LIBPATH="-L${PREFIX}/lib"
+export LIBRARY_PATH=${PREFIX}/lib
```

Comment:
There's no need to define any of these, as you're using conda's compilers. (It's possible that might break and you might need to re-add one or two of these, but I expect not.)

Reply:
Thanks! I'll remove them, and if something breaks, I'll re-add them as needed (I think the Mac OS X build was failing until I added `export LIBRARY_PATH=${PREFIX}/lib`)